lang/java/reef-runtime-local/src/main/java/org/apache/reef/runtime/local/driver/ResourceManager.java

```diff
@@ -65,27 +64,23 @@
   private final EventHandler<ResourceAllocationEvent> allocationHandler;
   private final ContainerManager theContainers;
   private final EventHandler<RuntimeStatusEvent> runtimeStatusHandlerEventHandler;
-  private final int defaultMemorySize;
-  private final int defaultNumberOfCores;
   private final ConfigurationSerializer configurationSerializer;
   private final RemoteManager remoteManager;
   private final REEFFileNames fileNames;
-  private final ClasspathProvider classpathProvider;
   private final double jvmHeapFactor;
   private final LoggingScopeFactory loggingScopeFactory;
 
   @Inject
   ResourceManager(
       final ContainerManager containerManager,
-      @Parameter(RuntimeParameters.ResourceAllocationHandler.class) final EventHandler<ResourceAllocationEvent> allocationHandler,
-      @Parameter(RuntimeParameters.RuntimeStatusHandler.class) final EventHandler<RuntimeStatusEvent> runtimeStatusHandlerEventHandler,
-      @Parameter(DefaultMemorySize.class) final int defaultMemorySize,
-      @Parameter(DefaultNumberOfCores.class) final int defaultNumberOfCores,
+      @Parameter(RuntimeParameters.ResourceAllocationHandler.class)
```

Comment:
Checkstyle problem, there was a line more than 120, that's why. Shall I change it back?